anuonifade/loanapp
# #150951802 & #150951892 & #153782932 User see the summary of contributions and loans on dashboard and admin user is redirected to admin dashboard

#### What does this PR do?
This PR redirects admin user to admin dashboard and displays the summary of all user's contributions and loans.

#### Description of task to be completed?
- As a logged in user, I should be able to see my total contributions, loan repayment (if any) (Functionality).
- As a user, I should be able to see my contribution and my total contribution so far. (Functionality).
- As an admin user, I should be redirected to the admin dashboard when I log in

#### How should this be manually tested?
- Run `rails db:migrate`
- Run `rails db:seed`  
- Login with `test-admin@nnpcloanapp.com` and password `password`
- You should be redirected to the admin dashboard

#### What are the relevant pivotal tracker stories?
#150951802 & #150951892 & #153782932

#### Screenshot
![screen shot 2017-12-19 at 8 37 45 pm](https://user-images.githubusercontent.com/26273055/34175396-9b25182a-e4fc-11e7-9785-a1a3fddbd680.png)
![screen shot 2017-12-19 at 8 38 12 pm](https://user-images.githubusercontent.com/26273055/34175397-9b4db2a8-e4fc-11e7-91e2-8387544ebc7f.png)
